### DISCUSSION TITLE
[2021.08.24] 고양이와 개는 몇 마리 있을까

### DISCUSSION THREAD

**Discussion by hyk339** [pyunji/Hyundai-ITnE-Team_QuadCore#34] [SQL]
[고양이와 개는 몇 마리 있을까](https://programmers.co.kr/learn/courses/30/lessons/59040)

**Comment by pyunji** [comment]
[고양이와 개는 몇 마리 있을까](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59040.sql)

**Comment by hyk339** [comment]
[고양이와 개는 몇 마리 있을까 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/commit/479a8dca7cef86df527363c62b0fe815c743a808)

**Comment by chaeyeon4u** [comment]
[고양이와 개는 몇 마리 있을까](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/chaeyeon/homework/SQL/PR_59040.sql)

**Comment by leechang4** [comment]
[고양이와 개는 몇 마리 있을까 solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR_59040.sql)
